index.js

```diff
@@ -1,43 +1,49 @@
 const http = require('http');
 const events = require('events');
-const {EventEmitter} = events;
+const { EventEmitter } = events;
 const express = require('express');
 const ws = require('ws');
 const engine = require('./build/Release/qr.node');
 
-async function start({
+async function initQr (qrEmitter, locomotionEmitter) {
+  const resultCode = engine.initVr();
+
+  // Reject if there was an error initializing OpenVR
+  // See https://github.com/ValveSoftware/openvr/wiki/HmdError for error codes
+  if (resultCode !== 0) {
+    console.error(resultCode === 100 ? 'OpenVR Installation not found' : 'Error initialising OpenVR', resultCode);
+    throw resultCode;
+  }
+
+  engine.createQrEngine(qrCode => qrEmitter.emit('qrCode', qrCode));
+
+  engine.createLocomotionEngine(locomotionInput =>
+    locomotionEmitter.emit('locomotionInput', locomotionInput)
+  );
+  engine.startThread();
+  return 0;
+}
+
+async function start ({
   port = 8000,
-  key,
+  key
```

Comment:
Trailing commas are very prevalent and preferred in Webaverse code so this rule will probably cause ongoing conflicts.

Reply:
I've removed the eslint config for now